README.rst

```diff
@@ -90,7 +90,64 @@ By default, blockstore is run as an app inside of Open edX. Enable it using the
 If you need to run blockstore as a separate service (e.g. for scalability or performance reasons), you can deploy blockstore in production using `the blockstore ansible role <https://github.com/openedx/configuration/tree/master/playbooks/roles/blockstore>`_.
 
 -------------------------------------------------------
-Running and testing as a separate service (development)
+Development in Devstack
+-------------------------------------------------------
+
+If you want to develop blockstore as a local python package installed in edx-platform, use the following steps:
+
+#. Prerequisite: Have an Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must use the Nutmeg release of Open edX (or newer) or be tracking the ``master`` branch of ``edx-platform``.
+
+#. To run blockstore as an app inside of Open edX, enable it using the waffle switch `blockstore.use_blockstore_app_api <https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-blockstore.use_blockstore_app_api>`_.
+
+#. Clone this repo in the ``src/`` directory, a sibling directory of your devstack folder.
+
+#. In your devstack directory, run the following commands in order:
+
+   .. code::
+      
+      # bring up the lms +studio containers
+      make dev.up.large-and-slow
+      # Bring up the studio shell
+      make dev.shell.studio
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+      
+      # do the same in the LMS
+      make dev.shell.lms
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+    
+      #restart the containers.
+      make dev.restart-devserver.lms dev.restart-devserver.studio
+
+      # you can see that your env now installed blockstore from a local directory by running:
+      make dev.shell.lms
+      pip list
+       
+      
+Seeing your changes will sometimes require running make requirements and then restarting the container.
+
+You're probably also going to want to create a collection for using content libraries, using:
+
+   .. code::
```

Comment:
What are your assumptions about the existence and state of a blockstore container prior to this point? What about now?

Reply:
I don't think it's superfluous to indicate that with Blockstore running this way a `docker ps` should not show any running blockstore container. While that instruction may be obvious to many, it will help some.